src/nm-config-data.h

```diff
@@ -178,6 +178,18 @@ int      nm_config_data_get_sriov_num_vfs (const NMConfigData *self, NMDevice *d
 
 NMGlobalDnsConfig *nm_config_data_get_global_dns_config (const NMConfigData *self);
 
+#define CON_DEFAULT_NOP(name) \
+	static char *NM_UNIQ_T (connection_default, NM_UNIQ) \
+		__attribute__((__used__, __section__("connection_defaults"))) = "" name
+
+#define CON_DEFAULT(name) \
```

Comment:
I think we should try that everything what we put in a header file has an "NM" prefix. In the past, we fixed a lot of offenders and do that nowadays quite consistently (still some exceptions). Anyway, can we call this `NM_CON_DEFAULT()` ?